install.sh

```diff
@@ -46,7 +46,9 @@ detect_platform() {
   platform="$(uname -s | tr '[:upper:]' '[:lower:]')"
 
   case "${platform}" in
-    linux) platform="linux" ;;
+    linux)
+      platform="linux"
+      getconf GNU_LIBC_VERSION >/dev/null 2>&1 || ldd --version >/dev/null 2>&1 || platform="linuxstatic";;
```

Comment:
Please define a function to determine the system is glibc-compatible

```suggestion
    linux)
      if is_glibc_system; then
        platform="linux"
      else
        platform="linuxstatic"
      fi
      ;;
```